cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -42,7 +46,7 @@ case class Workflow(
 case class CommandLineTool(
                             inputs: Inputs = Coproduct[Inputs](Array.empty[CommandInputParameter]),
                             outputs: Outputs = Coproduct[Outputs](Array.empty[CommandOutputParameter]),
-                            `class`: W.`"CommandLineTool"`.T = "CommandLineTool".narrow,
+                            `class`: CommandLineTool.`class`.type = CommandLineTool.`class`,
```

Comment:
Hm I tried to reply on my phone but apparently it didn't submit.  I think your workaround is fine given that I saw some weirdness too.